### ISSUE TITLE
Faceting thumbnails go mostly blank when visiting a visualization and making no changes

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/web-eda#718]
So make a visualisation with faceting (I used histogram, but I don't think it matters).

Then go back to the visualizations 'home' with the thumbnails.  It looks nice.

Then go back to the same visualization, and admire it some more.  **But don't make any modifications.**

Then go back to the thumbnails. It's just a shadow of its former self! :cry: 

![image](https://user-images.githubusercontent.com/308639/141854355-e71872b4-2d87-4971-8492-21bf55aff2b4.png)

**Comment by moontrip** [comment]
Actually I cannot reproduce this from my side. The only case when I can get an empty thumbnail from the pre-existing facet plot is I forcibly press minimize button or click Visualize tab before fully loading the plot, which I can understand the situation. 

A side note, LOL when I see the sentence "admire it some more" 😄

**Comment by bobular** [comment]
Oh, that may mean that it's browser specific of network delay related? :disappointed:

**Comment by jtlong3rd** [comment]
Aww, poor thumbnails! Using Bob's steps, I can reproduce with scatterplot (Mac with Chrome). 😢 

I'll check it out. 🔎

**Comment by aaronwlsong** [comment]
Looks good now, closing ticket.

---Here is how I replicated this issue-----------
Step1, establish the facet viz.
Study: Amazonia_Brazil_cohort, viz: Histogram, Main=Observation date,  overlay=ITN last night, facet=settlement 
 
![image](https://user-images.githubusercontent.com/61077845/142882301-d60f2b52-87f8-456b-ad96-8df248e4f31c.png)

Step2, click "Visualization"to go back the home page of viz.  as bobular said, it looks fine.
![image](https://user-images.githubusercontent.com/61077845/142882496-78cdf1bc-6e78-408f-b68a-2bcdd5ceda22.png)

Step3, Click the facet viz I created in step 1 without any modification, and then repeat step2 to go back viz homepage. Looks good. 
![image](https://user-images.githubusercontent.com/61077845/142883008-e1c66f07-91d9-4bfd-8612-65f9c6614a26.png)

Step4. Repeat this switch back and forth action several times, it still looks good. 

Also tested other studies and different plots, seems good too now on qa. website.
